README.md

```diff
@@ -6,51 +6,44 @@
 
 ## Description
 
-[MAVSDK](https://www.dronecode.org/sdk/) (previously known as "Dronecode SDK", "DroneCore", and "DroneLink") is an API and library for the [PX4 flight stack](http://github.com/PX4/Firmware) using [MAVLink](https://mavlink.io/en/).
-
-It is written in C++11 and aiming to be:
-
-- Easy to use with a simple API.
+[MAVSDK](https://mavsdk.mavlink.io/develop/en/) is a set of libraries providing a high-level API to [MAVLink](https://mavlink.io/en/).
+It aims to be:
+- Easy to use with a simple API supporting both synchronous (blocking) API calls and asynchronous API calls using callbacks.
 - Fast and lightweight.
 - Cross-platform (Linux, Mac, Windows, iOS, Android).
 - Extensible (using compile-time plugins).
+- Fully compliant with the MAVLink common microservices standard/definitions.
 
-## Interfacing
-
-The SDK currently takes care of the MAVLink messaging. Connections over serial, UDP, and TCP are supported on Linux, macOS, and Windows.
+The core library is written in C++ and is used as the common "backend" to auto-generated front end libraries for different programming languages. The generated libraries share similar APIs (because they are autogenerated from a common interface) but follow idiomatic patterns expected by end users (for example, the Python library is installed from PyPi using *pip*, and includes everything you need to get up and running).
 
-The library provides both synchronous (blocking) API calls, as well as asynchronous API calls using callbacks.
+This repo contains the source code for the C++ core. 
+Other important repos include:
+- [MAVSDK-Proto](https://github.com/mavlink/MAVSDK-Proto) - Collection of proto files used by gRPC in MAVSDK (common interface definitions)
+- [MAVSDK-Python](https://github.com/mavlink/MAVSDK-Python) - MAVSDK client for Python
+- [MAVSDK-Swift](https://github.com/mavlink/MAVSDK-Swift) - MAVSDK client for Swift.
+- [MAVSDK-Java](https://github.com/mavlink/MAVSDK-Java) - MAVSDK client for Java.
+- [MAVSDK-JavaScript](https://github.com/mavlink/MAVSDK-JavaScript) - JS Wrapper for MAVSDK using grpc-web to generate static http client compatible with Envoy Proxy
+- [Other repos](https://github.com/mavlink?q=MAVSDK) - Docs, examples, etc.
 
-## API Overview
 
-API consumers use `MAVSDK` class to discover and manage vehicles (`System` objects). Using the `System` object plugins such as e.g. `Action`, `Telemetry`, or `Mission` can be instantiated which provide information about the state of the drone and allow to interact with it.
+## Status
 
```

Comment:
@JonasVautherin Is the status of each of these correct? What dates should we put in for each of the libraries. I assumed "first usable version".

Reply:
I made some suggestions. Quick note: when we say "reliable and stable", we mean that it doesn't crash and can be used in production, right? As mentioned last week, the API is not completely stable yet, so (fairly small) changes may come and break the build.

Reply:
Yes, that is what I mean by "stable" in this case.